server/user.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @fileoverview user function handler
+ */
+var fs = require('fs');
+var childProcess = require('child_process');
+var readline = require('readline');
+var mkdirp = require('mkdirp');
+
+var utils = require('./utils.js');
+
+/** @type {user} */
+module.exports = user;
+
+/**
+ * @constructor
+ */
+function user() {}
+
+/** @enum {string} */
+user.QueryType = {
+  SIGHUP: 'sign-up',
+  SIGNIN: 'sign-in'
+};
+
+/**
+ * Name of user information file.
+ * @type {{
+ *   email: =string,
```

Comment:
I think there is no default value annotation (=) for record type. Use (string|undefined).